packages/code-highlight/src/languages/standard.ts

```diff
@@ -46,35 +88,99 @@ const standardLanguages: Record<string, LanguageFn> = {
   css,
   curl,
   dart,
+  diff,
+  docker: dockerfile,
+  dockerfile,
   elixir,
   go,
   graphql,
+  haskell,
   html: xml,
+  http,
+  ini,
   java,
   javascript,
   json,
   kotlin,
   less,
+  lua,
   makefile,
   markdown,
   matlab,
   nginx,
   objectivec,
   ocaml,
+  perl,
   php,
   plaintext,
   powershell,
+  properties,
   python,
   r,
   ruby,
   rust,
+  scala,
   scss,
   shell,
   sql,
   swift,
+  toml: ini,
   typescript,
   xml,
   yaml,
 }
+const mediumLanguages: Record<string, LanguageFn> = {
```

Comment:
can u add comments so future us knows what’s supposed to be in mediumLanguages, specializedLanguages and so on and what they are used for? 🙏